{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -154,7 +158,9 @@ system_requires = [
 system_runtime_requires = [
 {%- if cookiecutter.gui_framework == "Toga" %}
     # Needed to provide GTK
-    "gtk",
+    "gtk3",
+    # Needed to provide PyGObject bindings
+    "gobject-introspection-runtime",
     # Needed to provide WebKit2 at runtime
     # "webkit2gtk",
 {%- elif cookiecutter.gui_framework == "PySide6" %}
```

Comment:
cc: @proneon267

I needed `qt6-base` at runtime for the PySide6 helloworld app. Although, I think it also required me to list it as a build time requirement....I'm not sure if this is avoidable in this configuration somehow.

Reply:
Yeah - it appears that Arch needs all the requires dependencies to be present on the build machine, just so it can verify that the package exists. I'm on the border of modifying the system packages check in Briefcase to include `system_runtime_requires` on arch...